src/input_xml.F90

```diff
@@ -975,6 +975,13 @@ subroutine read_geometry_xml()
       else
         call fatal_error("Must specify id of cell in geometry XML file.")
       end if
+
+      ! Copy cell name
+      c % name = ''
```

Comment:
I think that I just did this since that's how it was previously done for the tally labels. However, I just went through and removed all of these empty string initialization lines.